src/inventory-brand/DTOs/BrandDTO.ts

```diff
@@ -0,0 +1,7 @@
+import { ObjectId } from "mongodb";
+
+
+export type BrandDTO = {
+    userID: ObjectId;
+    name: String;
```

Comment:
this shoulde be string not String. "String" is a string constructor. "string" is a type.